pkg/northbound/gnmi/v2/get.go

```diff
@@ -138,7 +138,8 @@ func (s *Server) processRequest(ctx context.Context, req *gnmi.GetRequest, group
 			}
 		}
 
-		updates, err := s.getUpdate(ctx, targets[targetID], prefix, &pathInfo{}, req.GetEncoding(), groups)
+		prefixRegexp := utils.StrPath(prefix)
```

Comment:
I found the name `prefixRegexp` a bit confusing - would `prefixPath` suit its content a bit more